src/blocks/opentable/save.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies.
+ */
+const {__} = wp.i18n;
+
+const OpenTableSave = function( props ) {
+	const {
+		attributes: {
+			restaurantId,
+			dateLabel,
+			timeLabel,
+			partySizeLabel,
+			submitButtonText
+		},
+	} = props;
+
+	const partySize = ( Array.from( Array( 11 ).keys() ) ).slice( 1 );
+
+	const partySizeOptions = partySize.map( option => <option value={option}> {option} </option> );
```

Comment:
move `partySizeOptions` and `arrayTimeOptions` in `./lib/settings.php` and use `./components/with-settings` to retrieve these setttings

Reply:
We are not adding an option for party size and reservation time. We will let those values that are defaults in OpenTable.